### PR TITLE
MAINT: update citations

### LINKED ISSUE #143: Update citation information
The README and provenance citations should be updated to cite the Bioinformatics article instead of the pre-print.

https://doi.org/10.1093/bioinformatics/btac639

### PR DESCRIPTION
**Pull request by misialq**
Closes #143.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,7 +205,7 @@ Problem? Suggestion? Technical errors and user support requests can be filed on
 If you use `fondue` in your research, please cite the following:
 
 Michal Ziemski, Anja Adamov, Lina Kim, Lena Flörl, Nicholas A. Bokulich. 2022. Reproducible acquisition, management, and meta-analysis of nucleotide sequence (meta)data using q2-fondue.
-bioRxiv 2022.03.22.485322; doi: https://doi.org/10.1101/2022.03.22.485322
+_Bioinformatics_, 2022; doi: https://doi.org/10.1093/bioinformatics/btac639
```
Comment (nbokulich) [inline README.md:208]: the year should be removed on this line (it is already on line 207)

File: q2_fondue/citations.bib [hunk 2]
```diff
@@ -1,10 +1,12 @@
 @article {Ziemski2022,
-	author = {Ziemski, Michal and Adamov, Anja and Kim, Lina and Fl{\"o}rl, Lena and Bokulich, Nicholas A.},
+	author = {Ziemski, Michal and Adamov, Anja and Kim, Lina and Flörl, Lena and Bokulich, Nicholas A},
 	title = {Reproducible acquisition, management, and meta-analysis of nucleotide sequence (meta)data using q2-fondue},
 	year = {2022},
-	doi = {10.1101/2022.03.22.485322},
-	URL = {https://www.biorxiv.org/content/early/2022/03/25/2022.03.22.485322},
-	journal = {bioRxiv}
+    month = {09},
```
Comment (nbokulich) [inline q2_fondue/citations.bib:5]: this line should be indented or removed. It is out of alignment with the other lines

File: q2_fondue/citations.bib [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
+	doi = {10.1093/bioinformatics/btac639},
+	URL = {https://doi.org/10.1093/bioinformatics/btac639},
+	journal = {Bioinformatics},
+    issn = {1367-4803},
```
Comment (nbokulich) [inline q2_fondue/citations.bib:9]: also out of alignment

**Review by nbokulich — CHANGES_REQUESTED** [review]
thanks @misialq ! some super minor comments in-line

File: q2_fondue/citations.bib (hunk 2 above)
Comment (misialq) [reply q2_fondue/citations.bib]: It seems like the indents were messed up a bit throughout the file so I just reindented everything. Hope it's fine now.

**Pull request merged**